arches/app/templates/views/components/search/standard-search-view.htm

```diff
@@ -2,148 +2,157 @@
 {% load template_tags %}
 {% load static %}
 
-<article class="main-search-container content-panel flexrow" data-bind="click:function() { sharedStateObject.menuActive(false); }, visible: true", style="display:none;" aria-label="{% trans 'Search page contents' %}">
-        <section class="flex search-results-panel" aria-label="{% trans 'Search results' %}">
-            <div class="search-attribute-widget">
-                <div class="search-filter-tools-container">
-                    <div data-bind="component: {
-                        name: 'term-filter',
-                        params: sharedStateObject
-                    }"></div>
-                    <div class="search-inline-filters-row">
-                        <button class="btn-sm btn btn-mint btn-labeled clear-filter search-btn" data-bind="click: function() {clearQuery();}"><span>{% trans "Clear Filters" %}</span></button>
-                        <div class="search-inline-filters-btns">
-                            <div class="qa-filter" data-bind="visible: sharedStateObject.userIsReviewer() !== false,
-                                component: {
-                                    name: 'provisional-filter',
-                                    params: sharedStateObject
-                                }">
-                            </div>
-                            <div class="resource-filter" data-bind="component: {
-                                name: 'resource-type-filter',
-                                params: sharedStateObject
-                            }">
-                        </div>
-                        </div>
-                    </div>
-                </div>
-                <div class="search-sort-tools-container">
-                    <div class="search-count-container">
-                        <p class="search-title" aria-live="assertive" data-bind="text: $root.translations.shownTotalSearchResults(hits(), total())"></p>
-                    </div>
-                    <div data-bind="component: {
-                        name: 'sort-results',
+<article 
+    class="main-search-container content-panel flexrow" 
+    data-bind="
+        click: function() { 
+            sharedStateObject.menuActive(false); 
+            return true;
+        }, 
+        visible: true
+    " 
```

Comment:
I think some of the formatting changes here https://github.com/archesproject/arches/commit/f4579edae769bbdf32e6dc48b76ba80ca786225c might be making the merge a bit tricky. It might make sense just to accept the code in v8, and then repeat the work done in f4579ed.

Reply:
@chiatt I just pushed a fix for that